cardano-api/src/Cardano/Api/Address.hs

```diff
@@ -554,6 +559,18 @@ fromShelleyStakeCredential (Shelley.KeyHashObj kh) =
 fromShelleyStakeCredential (Shelley.ScriptHashObj sh) =
     StakeCredentialByScript (fromShelleyScriptHash sh)
 
+fromShelleyPaymentCredential ::
+  Shelley.PaymentCredential StandardShelley -> PaymentCredential
+fromShelleyPaymentCredential = \case
+  Shelley.KeyHashObj    kh -> PaymentCredentialByKey (PaymentKeyHash kh)
+  Shelley.ScriptHashObj sh -> PaymentCredentialByScript (ScriptHash sh)
+
+fromShelleyStakeReference ::
+  Shelley.StakeReference StandardShelley -> StakeAddressReference
+fromShelleyStakeReference = \case
+  Shelley.StakeRefBase stakecred -> StakeAddressByValue (fromShelleyStakeCredential stakecred)
+  Shelley.StakeRefPtr ptr -> StakeAddressByPointer ptr
+  Shelley.StakeRefNull -> NoStakeAddress
```

Comment:
Can we follow the local style (e.g. the functions immediately above) and use
```
foo (Shelley.Blah ..) = Blah ...
```
rather than lambda case.

Reply:
Done.